grafana/README.md

```diff
@@ -26,6 +26,9 @@ And don't forget to include the generated `risingwave-<xxx>-dashboard.json` in t
 ## Update without Restarting Grafana
 
 ```bash
+./risedev configure # make sure that grafana is turned on
+./risedev dev full
```

Comment:
I guess `update` means there already has been one grafana running 🤔 no need to call `risedev full` to start a new cluster.

Reply:
Will undo this change. I added it here, because I did not have a cluster running 😁